cookbook/agent_os/a2a_client/01_basic_messaging.py

```diff
@@ -0,0 +1,66 @@
+"""
+Basic A2A Messaging with A2AClient
+
+This example demonstrates simple message sending and receiving
+using the A2A protocol.
+
+Prerequisites:
+1. Start an AgentOS server with A2A interface:
+   python cookbook/agent_os/interfaces/a2a/basic.py
+
+2. Run this script:
+   python cookbook/agent_os/a2a_client/01_basic_messaging.py
```

Comment:
Lets organize the cookbooks as `/clients/os/...` and `/clients/a2a/...`, so we should move it out of the agent_os directory